deploy/env/e2e-tests.yaml

```diff
@@ -1,3 +1,2 @@
 member-operator:
-  identity-provider: 'testIdP'
```

Comment:
Correct me if I'm wrong, but you should not remove these values from the env config files. This is btw the reason why your e2e tests are failing.
I understand that you need to primarily use the value from the ConfigMap, but you can still combine them and replace it with the value from CM, right?
Or, you could rename it to `default-identity-provider` and use this value as the default one if no value is specified in the CM.

Reply:
@MatousJobanek shouldn't the default be set via viper? the same way we do for other values that are being set via the configMap?

Reply:
It's up to us how we deal with the default values - if we use viper functionality or some other way...
I see that you changed the IdP name to `rhd` for the e2e tests - this is a feasible solution as well and much easier :+1: